solution/ui/e2e/cypress/e2e/api.spec.cy.js

```diff
@@ -85,18 +83,3 @@ describe("Synonyms endpoint special character testing", () => {
         });
     });
 });
-
-// https://ahmed-alsaab.medium.com/cypress-api-schema-validation-with-ajv-ec61348374f7
-describe("Schema validation", () => {
-    describe("Search.gov Resources", () => {
-        it("gets successful response and expected schema from v3/resources/search", () => {
-            cy.getSearchGovResources(SEARCH_TERM).then((response) => {
-                expect(response.status).to.eq(200);
-                expect(response.isOkStatusCode).to.eq(true);
-                expect(response.statusText).to.eq("OK");
-
-                cy.validateSchema(getSearchGovResourcesSchema, response.body);
```

Comment:
We no longer call out to any external APIs for data, so no more need to validate the schema of any API response.  We're still going to keep around the `validateSchema` method in case we need it later.